app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/NoCollectionsMessageViewHolder.kt

```diff
@@ -5,59 +5,59 @@
+import org.mozilla.fenix.compose.ComposeViewHolder
 import org.mozilla.fenix.home.sessioncontrol.CollectionInteractor
-import org.mozilla.fenix.utils.view.ViewHolder
 
-@OptIn(ExperimentalCoroutinesApi::class)
-open class NoCollectionsMessageViewHolder(
-    view: View,
+/**
+ * [RecyclerView.ComposeViewHolder] for displaying a message detailing the collections feature and
+ * allowing users to easily start creating their first.
+ *
+ * @param composeView [ComposeView] which will be populated with Jetpack Compose UI content.
+ * @param viewLifecycleOwner [LifecycleOwner] to which this Composable will be tied to.
+ * @param store to receive live updates to normal tabs.
+ * @param interactor [CollectionInteractor] callback for user interaction.
+ */
+class NoCollectionsMessageViewHolder(
+    composeView: ComposeView,
     viewLifecycleOwner: LifecycleOwner,
-    store: BrowserStore,
-    interactor: CollectionInteractor
-) : ViewHolder(view) {
+    private val store: BrowserStore,
+    private val interactor: CollectionInteractor
+) : ComposeViewHolder(composeView, viewLifecycleOwner) {
 
     init {
-        val binding = NoCollectionsMessageBinding.bind(view)
-
-        binding.addTabsToCollectionsButton.apply {
-
-            setOnClickListener {
-                interactor.onAddTabsToCollectionTapped()
-            }
-            isVisible = store.state.normalTabs.isNotEmpty()
-        }
-
-        binding.removeCollectionPlaceholder.apply {
-            increaseTapArea(
-                view.resources.getDimensionPixelSize(R.dimen.tap_increase_16)
-            )
-            setOnClickListener {
-                interactor.onRemoveCollectionsPlaceholder()
-            }
-        }
-
-        store.flowScoped(viewLifecycleOwner) { flow ->
-            flow.map { state -> state.normalTabs.size }
-                .ifChanged()
-                .collect { tabs ->
-                    binding.addTabsToCollectionsButton.isVisible = tabs > 0
-                }
-        }
+        val horizontalPadding =
+            composeView.resources.getDimensionPixelSize(R.dimen.home_item_horizontal_margin)
+        val verticalPadding =
+            composeView.resources.getDimensionPixelSize(R.dimen.home_fragment_top_toolbar_header_margin)
+        composeView.setPadding(horizontalPadding, verticalPadding, horizontalPadding, 0)
     }
 
     companion object {
-        const val LAYOUT_ID = R.layout.no_collections_message
+        val LAYOUT_ID = View.generateViewId()
+    }
+
+    @Composable
+    override fun Content() {
+        val normalTabsState = store.observeAsComposableState { state -> state.normalTabs }.value ?: emptyList()
+
+        Spacer(Modifier.height(40.dp))
+
+        CollectionsPlaceholder(
+            showAddToCollectionButton = normalTabsState.isNotEmpty(),
+            onAddTabsToCollectionButtonClick = interactor::onAddTabsToCollectionTapped,
+            onRemoveMessageClick = interactor::onRemoveCollectionsPlaceholder
+        )
```

Comment:
I think you need to add `Spacer(Modifier.height(12.dp))` after this https://github.com/mozilla-mobile/fenix/blob/44fa89915467745af4ebf20fde1ab2eddac15411/app/src/main/res/layout/no_collections_message.xml#L12